tests/OpenTK.Tests.Integration/GameWindowTests.fs

```diff
@@ -0,0 +1,194 @@
+﻿namespace OpenTK.Tests.Integration
+
+open Xunit
+open FsCheck
+open FsCheck.Xunit
+open System
+open System.Runtime.InteropServices
+open OpenTK
+
+module GameWindow =
+    module General =
+        [<Fact>]
+        let ``Can create and close GameWindow`` () =
+            let gw = new OpenTK.GameWindow()
```

Comment:
GameWindow is disposable, so let's `use` instead of `let`. Will most likely help performance for larger test sets and won't hurt.